base/src/Data/Macaw/AbsDomain/AbsState.hs

```diff
@@ -256,31 +256,31 @@ instance EqF (AbsValue w) where
 instance MemWidth w => Show (AbsValue w tp) where
   show = show . pretty
 
-ppSet :: [Doc] -> Doc
+ppSet :: [Doc ann] -> Doc ann
 ppSet = encloseSep lbrace rbrace comma
 
 instance MemWidth w => Pretty (AbsValue w tp) where
-  pretty (BoolConst b) = text (show b)
-  pretty (FinSet s) = text "finset" <+> ppIntegerSet s
-  pretty (CodePointers s b) = text "code" <+> ppSet (s0 ++ sd)
-    where s0 = if b then [text "0"] else []
+  pretty (BoolConst b) = viaShow b
+  pretty (FinSet s) = pretty "finset" <+> ppIntegerSet s
```

Comment:
I'm inclined to just rely on `OverloadedStrings` and use `"finset"` since it looks like `prettyprinter` `Doc` implements `IsString`.

Reply:
Yes, it does implement `IsString`. My default approach has been to change `text "foo"` to `"foo"` if the module already had `OverloadedStrings` enabled, and to use `pretty "foo"` otherwise. But I can certainly go back and enable `OverloadedStrings` in those modules instead.

Reply:
Thanks.  Enabling `OverloadedStrings` would be my preference.

Reply:
I have a commit that enables `OverloadedStrings` on a separate branch (`doc-overloaded-strings`). I just wanted to get CI to pass before I cherry-pick that commit back onto this PR.